### ISSUE TITLE
Hyperspectra: Treat small number of features as Parallel coordinates plot

### ISSUE THREAD

**Issue by stuart-cls** [Quasars/orange-spectroscopy#488]
This often comes up as users work-around #330 after using a File -> Integrate (output as features) -> Hyperspectra workflow.

Although mitigated by #482 , I've often thought that having the reduced variables (peak area, PCA, etc) plotted along the spectra sub-widget gives a nice parallel coordinates plot. It's only missing per-feature labels (which means in practice users try to remember the order of their features, which is unpleasant).

So basically: for datasets with small numbers of features (say 10), we could label the x-axis categorically instead of as an index-based continuum.
